components/brave_wallet_ui/utils/datetime-utils.ts

```diff
@@ -48,6 +48,32 @@ export function formatDateAsRelative (date: Date, now: Date = new Date()) {
  * @param {SerializableTimeDelta} timeDelta
  * @return {!Date}
  */
- export function serializedTimeDeltaToJSDate (timeDelta: SerializableTimeDelta): Date {
+export function serializedTimeDeltaToJSDate (
+  timeDelta: SerializableTimeDelta
+): Date {
   return new Date(Number(timeDelta.microseconds) / 1000)
 }
+
+export const formatTimelineDate = (date: Date) => {
+  const formatedDate =
+    new Date(date)
+      .toLocaleDateString(
+        'en-US',
```

Comment:
Shouldn't this be based on the user's locale? For example, `mm/dd/yyyy` is used only in the US.